lib/methods/describe.js

```diff
@@ -48,8 +48,32 @@ exports.describe = function describe() {
 		 * @returns {*}
 		 */
 		action: function describedAction(req, resp, next) {
+			/**
+			 * Sends the result of POST request to the response.
+			 * @param err
+		 	 * @param instance
+			 * @returns {*}
+			 */
+			var createdResponse = function (err, instance) {
+				if (err) {
+					return next(err);
+				}
+				resp.response.set('Location', req.path + instance.getPrimaryKey());
+				resp.response.send(201);
+				next();
+			};
+
 			try {
-				resp.stream(context.model[context.methodName], req.params, next);
```

Comment:
I don't think we should replace the `res.stream` here and rather also do the transformation for POST requests responses in the `handleResponse` function. That way the user retains complete control about what to do with the response from his API.